src/main/java/com/nerdtranslator/lingueeapibridge/service/impl/TextToSpeechServiceImpl.java

```diff
@@ -1,18 +1,76 @@
 package com.nerdtranslator.lingueeapibridge.service.impl;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.api.gax.core.CredentialsProvider;
+import com.google.auth.oauth2.ServiceAccountCredentials;
 import com.google.cloud.texttospeech.v1.*;
 import com.google.protobuf.ByteString;
 import com.nerdtranslator.lingueeapibridge.service.TextToSpeechService;
 import org.springframework.stereotype.Service;
 
+import java.io.ByteArrayInputStream;
+import java.io.FileInputStream;
 import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Properties;
 
 @Service
 public class TextToSpeechServiceImpl implements TextToSpeechService {
 
+    @Override
+    public byte[] transformTextToSound(String textToTransfer, String langCode) {
+        return getSpeechFromText(textToTransfer, langCode);
+    }
+
+    private static Map<String, String> getCredentials() {
+        Properties properties = new Properties();
+        Map<String, String> credentials = new LinkedHashMap<>();
+
+        try (FileInputStream inputStream = new FileInputStream("src/main/resources/sensitive.properties")) {
```

Comment:
Don't read credentials straight from sensitive.properties, add variables as link in application.properties, inject them using Value annotation